### PR TITLE
Quick reporting fixes

### PR DESCRIPTION
**Pull request by Index01**
- top level reporting now in function
    - diff re-written and called on guest list upload
    - email functional

### CODE REVIEW

File: backend/createStaffAndRooms.py [hunk 1]
```diff
@@ -118,28 +114,32 @@ def create_staff(init_file=None):
...
         staff.save()
-    
+
         print(f"[+] Created staff: {staff_new['name']}, {staff_new['email']}, otp: {otp}, isadmin: {staff_new['is_admin']}")
+        print(SEND_MAIL)
         if(SEND_MAIL=="True"):
             print(f'[+] Sending invite for staff member {staff_new["email"]}')
-    
+            apppass = os.environ['ROOMBAHT_EMAIL_HOST_PASSWORD']
+            print(f'pass {apppass}')
```
Comment (otakup0pe) [inline backend/createStaffAndRooms.py:123]: gonna be removing this when i take a pass through the logging heh

File: backend/createStaffAndRooms.py [hunk 2]
```diff
@@ -118,28 +114,32 @@ def create_staff(init_file=None):
...
+
             body_text = f"""
                 Congratulations, u have been deemed Staff worthy material.
 
                 Email {staff_new['email']}
                 Admin {otp}
-                
-                login at blahblahblah/login  
+
+                login at blahblahblah/login
```
Comment (otakup0pe) [inline backend/createStaffAndRooms.py:132]: Do we need to do anything other than pass in the `ROOMBAHT_HOST` env var in order to generate the real url?

File: backend/reservations/views/admin.py [hunk 3]
```diff
@@ -130,7 +132,7 @@ def guest_contact_new(guest_new, otp):
 This is your password, there are many like it but this one is yours. Once you use this password on a device, RoomBaht will remember you, but only on that device.
 Copy and paste this password. Because let’s face it, no one should trust humans to make passwords:
 {otp}
-http://ec2-3-21-92-196.us-east-2.compute.amazonaws.com:3000/login
+http://rooms.take3presents.com/login
```
Comment (otakup0pe) [inline backend/reservations/views/admin.py:135]: See above comment about `ROOMBAHT_HOST` env var.

File: samples/exampleMainGuestList.csv [hunk 4]
```diff
@@ -11,3 +11,4 @@ cqwas43,cc00ff88,Index,tyler32bit@gmail.com,Babyface Suite (1 King Bed)
 cq12k43,cc00ff88,Index,tyler32bit@gmail.com,Babyface Suite (1 King Bed)
 iamanid,Pez,Mike,mpesaven@gmail.com,Babyface Suite (1 King Bed)
 notanid,Pez,Mike,mpesaven@gmail.com,Babyface Suite (1 King Bed)
+bizbaz,Gadget,Hiiam,jonafree@gmail.com,Knew Management Suite (1 King Bed)
```
Comment (otakup0pe) [inline samples/exampleMainGuestList.csv:14]: Ah - makes sense we need to have these in multiple files. We might want to update the `README` with some more details on the data format(s).

**Review by otakup0pe — APPROVED** [review]
Several comments but nothing blocking. Mostly placeholders for future work around

* logging
* documentation change
* less hard coding

File: backend/config/roombaht-systemd.conf [hunk 5]
```diff
@@ -14,13 +14,14 @@ Type=notify
 NotifyAccess=all
 StandardError=syslog
 RuntimeDirectory=uwsgi
-ExecStart=/opt/roombaht-backend/.env/bin/uwsgi --ini /opt/roombaht-backend/config/roombaht-wsgi.ini
+ExecStart=/opt/roombaht-backend/venv/bin/uwsgi --ini /opt/roombaht-backend/config/roombaht-wsgi.ini
```
[the remark this answers is not in the document]
Comment (Index01) [reply backend/config/roombaht-systemd.conf:17]: I think @mpesavento changed the venv name to make it less confusing with `.env` file

File: backend/reservations/views/admin.py (hunk 3 above)
Comment (Index01) [reply backend/reservations/views/admin.py:135]: Deffff

File: samples/exampleMainGuestList.csv (hunk 4 above)
Comment (Index01) [reply samples/exampleMainGuestList.csv:14]: Yea we need to verify this format with the latest secret party exports

**Pull request merged**